docs/service-brokers/helm-broker-service-classes/azure-broker/docs/plans-details.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Services and Plans
+type: Details
+---
+
+## Service description
+
+The `Azure Broker` service provides the following plan names and descriptions:
+
+| Plan Name | Description |
+|-----------|-------------|
+| `Default` | Installs the Azure Broker in a default configuration. |
+
+## Provisioning
```

Comment:
Delete this section and leave only the ## Provisioning parameters. The info in those two sections is doubled.

Reply:
discussed and fixed